libr/core/cbin.c

```diff
@@ -22,33 +22,40 @@
 // dup from cmd_info
 #define PAIR_WIDTH 9
 
-static void pair(const char *key, const char *val, int mode, bool last) {
-	if (!val || !*val) {
+static void pair(const char *key, const char *val_norm, const char *val_json, int mode, bool last) {
```

Comment:
why not just filter the json in here instead of passing 2 args?

Reply:
uh ok 89d55249208f289ddee2fb0548c48bd55c8dcf56

Reply:
89d55249208f289ddee2fb0548c48bd55c8dcf56 -> a433ed87cdb6bc2634d515d120b67b3581f35e9f due to a git rebase